sympy/printing/tests/test_latex.py

```diff
@@ -1226,13 +1226,13 @@ def test_matMul():
     l = LatexPrinter()
     assert l._print_MatMul(2*A) == '2 A'
     assert l._print_MatMul(2*x*A) == '2 x A'
-    assert l._print_MatMul(-2*A) == '-2 A'
+    assert l._print_MatMul(-2*A) == '- 2 A'
     assert l._print_MatMul(1.5*A) == '1.5 A'
     assert l._print_MatMul(sqrt(2)*A) == r'\sqrt{2} A'
     assert l._print_MatMul(-sqrt(2)*A) == r'- \sqrt{2} A'
-    assert l._print_MatMul(2*sqrt(2)*x*A) == r'2 \sqrt{2} x A'
-    assert l._print_MatMul(-2*A*(A + 2*B)) in [r'-2 A \left(A + 2 B\right)',
-        r'-2 A \left(2 B + A\right)']
+    assert l._print_MatMul(2*sqrt(2)*x*A) == r'2 x \sqrt{2} A'
```

Comment:
Why did this change?

Reply:
It keeps the args order of `MatMul(Mul(2, x, sqrt(2)), A)`:
```python
In [1]: 2*x*sqrt(2)
Out[1]: 2⋅√2⋅x

In [2]: expr = 2*x*sqrt(2)

In [3]: expr
Out[3]: 2⋅√2⋅x

In [4]: expr.args
Out[4]: (2, x, √2)
```

Ideally `_print_MatMul` should be abolished in favor of `_print_Mul`, now that `MatMul` is subclassing `Mul`. Unfortunately `MatMul` should not be represented by fractions, so maybe it's better to keep things separated.

Reply:
I guess it should reuse the sorting that _print_Mul uses.